index.html

```diff
@@ -52,11 +52,241 @@ <h2>I’am a Software Developer</h2>
                     </a>
                 </li>
             </ul>
-            <div class="after-socials">
 
-            </div>
 
         </section>
+        <section class="works-main-container">
+            <section class="recent-works">
+                <h2 class="recent-works-title">My Recent Works</h2>
+                <section class="cards-container">
+                    <!-- First Card -->
+                    <div class="card">
+                        <img src="./images/card1.png" alt="card-1" class="card-1-image">
+                        <div class="card-description">
+                            <h2 class="card-title">Multi-Post Stories</h2>
+                            <p class="supporting-text">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                been the industry's standard
+                                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
```

Comment:
| Figma design | Your design |
|-------------------|------------------|
| ![Screenshot from 2022-11-05 11-34-10](https://user-images.githubusercontent.com/65129228/200112307-67bfc755-9490-4654-be9c-7f5fec5d10ce.png) | ![Screenshot from 2022-11-05 11-44-58](https://user-images.githubusercontent.com/65129228/200112329-03d12bf1-cbf1-4b36-a4f1-4356e61c575f.png) |

- [ ] Great job on implementing the design for the work section :100: , but some issues to fix in the alignment of the items.


| Figma design | Your design |
|-------------------|------------------|
| ![Screenshot from 2022-11-05 11-47-14](https://user-images.githubusercontent.com/65129228/200112455-0bc36e1a-8fee-4f53-a86a-be41b86c9f55.png) | ![Screenshot from 2022-11-05 12-20-23](https://user-images.githubusercontent.com/65129228/200112838-f3c83b5b-b09a-41d2-bf52-545a2756c393.png) |

- [ ] Kindly stick with the Figma template design you chose; make sure the button container's right margin is equivalent to the Figma template design. Fix that :+1:

Reply:
Fixed.